modules/arm_plugin/src/arm_converter/arm_converter_reduce_ops.cpp

```diff
@@ -7,11 +7,12 @@
 #include <arm_compute/runtime/NEON/functions/NEReduceMean.h>
 #include <ngraph/runtime/reference/logical_reduction.hpp>
 #include "arm_converter/arm_converter.hpp"
+#include "opset/utils.hpp"
 
 namespace ArmPlugin {
 template<typename Reduce>
 static auto ConvertReduce(const Reduce& node, const arm_compute::ReductionOperation& op, Converter* converter) {
-    auto axes = dynamic_cast<const opset::Constant&>(*(node.input_value(1).get_node())).cast_vector<int64_t>();
+    auto axes = dynamic_cast<opset::Constant&>(*(node.input_value(1).get_node())).cast_vector<std::int64_t>();
```

Comment:
Old casting approach retained

Reply:
Due to some pitfalls in `ov::as_type_ptr` I have to use `dynamic_cast` here. But as it is uses reference cast it is safe enough but the error will be not so readable.